packages/utils/src/useColumnManager/useColumnManager.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import get from 'lodash/get';
+import ColumnManager from '@redhat-cloud-services/frontend-components/ColumnManager';
+import { filterColumnsBySelected, filterManageableColumns } from './helper';
+
+const useColumnManager = (columns = [], options = {}) => {
+  const { columnManagerSelectProp: selectProp = 'key', manageColumns: enableColumnManager } = options;
+  const managableColumns = filterManageableColumns(columns, selectProp);
+  const [selectedColumns, setSelectedColumns] = useState(managableColumns.map((column) => get(column, selectProp)));
+  const [isManagerOpen, setIsManagerOpen] = useState(false);
+
+  const onClick = () => {
+    setIsManagerOpen(true);
+  };
+
+  const onSave = (newSelectedColumns) => {
+    setSelectedColumns(newSelectedColumns);
+    setIsManagerOpen(false);
+  };
+
+  return enableColumnManager && managableColumns.length !== 0
```

Comment:
You can use safe accessor here
```suggestion
  return managableColumns?.length !== 0
```